roles/deploy/tasks/main.yml

```diff
@@ -27,6 +27,11 @@
       repo: "{{ repo }}"
       dest: /home/{{ ansible_ssh_user }}/{{ project_name }}
 
+- name: install sqlite dependencies
```

Comment:
I thought about a debian role, but I think this would be necessary in a pristine ubuntu build as well.